README.md

```diff
@@ -9,6 +9,7 @@ Terraform module which provisions an addon ([Helm release](https://registry.terr
 ```hcl
 module "eks_blueprints_addon" {
   source = "aws-ia/eks-blueprints-addon/aws"
+  version = "1.0.0"
```

Comment:
this will quickly become out of sync, theres no mechanism to keep it in sync. just like helm chart install examples, they don't specify the version - its up to users to select the version

Reply:
Yeah I hear that but looking to avoid breaking changes and dependencies again. Perhaps a comment stating to update the version?

Reply:
Added comment

Reply:
we can't control whether or not users pin the versions. the unpinned version issue we faced was self-inflicted and internal to our addons, not something users were doing incorrectly